spec/use-hotkeys/use_log_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from 'stimulus'
+import { bindHotkeys, unbindHotkeys } from '../../src'
+
+export default class UseLogController extends Controller {
+  static hotkeys = { '/': 'singleKeyHandler', 'cmd+a': 'metaKeyHandler' }
```

Comment:
It could make sense to think about a way to declare [options](https://github.com/jaywcjlove/hotkeys#option)